crates/bevy_pbr/src/deferred/mod.rs

```diff
@@ -408,14 +426,44 @@ pub fn prepare_deferred_lighting_pipelines(
             Option<&EnvironmentMapLight>,
             Option<&ShadowFilteringMethod>,
             Option<&ScreenSpaceAmbientOcclusionSettings>,
+            (
+                Option<&NormalPrepass>,
```

Comment:
I guess it's a bit out of scope but you could use `Has<XPrepass>` here instead of `Option`. I believe it might be a bit more efficient.

Reply:
Same comment applies to pretty much all the queries with `Option<XPrepass>` in this PR

Reply:
Made the change!